src/modules/endpoint/sagas/loadCityContent.js

```diff
@@ -54,16 +56,24 @@ export default function * loadCityContent (
   yield put(pushLanguages)
 
   if (languages.map(language => language.code).includes(newLanguage)) {
-    const [categoryUrls, eventUrls] = yield all([
-      call(loadCategories, newCity, newLanguage, dataContainer, shouldUpdate),
-      call(loadEvents, newCity, newLanguage, dataContainer, shouldUpdate)
+    const [categoriesMap, events] = yield all([
+      call(loadCategories, newCity, newLanguage, dataContainer, shouldUpdate, shouldRefreshResources),
+      call(loadEvents, newCity, newLanguage, dataContainer, shouldUpdate, shouldRefreshResources)
```

Comment:
As far as I can see, the `shouldRefreshResources` is never used in `loadCategories´ and `loadEvents`, so it should probably be removed from the call.

Reply:
Whoops, yeah. That's a left-over. Thanks :)